pantheon/src/test/java/tech/pegasys/pantheon/util/PermissioningConfigurationValidatorTest.java

```diff
@@ -69,7 +69,17 @@ public void nodesWhitelistOptionWhichDoesNotIncludeBootnodesMustError() throws E
           ethNetworkConfig, permissioningConfiguration);
       fail("expected exception because ropsten bootnodes are not in node-whitelist");
     } catch (Exception e) {
-      assertThat(e.getMessage().startsWith("Bootnode")).isTrue();
+      assertThat(e.getMessage().startsWith("Bootnode(s) not in nodes-whitelist")).isTrue();
```

Comment:
Could it be simplified like:
`assertThat(e.getMessage()).startsWith("Bootnode(s) not in nodes-whitelist")`?

Reply:
👍